app/resources/base.html.template

```diff
@@ -17,7 +17,9 @@
 
 <!DOCTYPE html>
 <html dir="{{env.rtl|yesno:"rtl,ltr"}}" lang="{{env.lang}}">
-<head>{% block head %}
+<head>
+  <meta charset="utf-8">
```

Comment:
That sounds like a good practice in general, so you can do the same here, but can you still put it inside {% block head %}?:

```
<head>
  {% block head %}
    <meta charset="utf-8">
    ...
```

It would be easier to understand if {% block head %} contains the whole content of the `<head>` tag.